test/test_client_context.py

```diff
@@ -30,6 +30,15 @@ def test_must_connect(self):
                         'PYMONGO_MUST_CONNECT is set. Failed attempts:\n%s' %
                         (client_context.connection_attempt_info(),))
 
+    def test_serverless(self):
+        if 'TEST_SERVERLESS' not in os.environ:
+            raise SkipTest('TEST_SERVERLESS is not set')
+
+        self.assertTrue(client_context.connected,
```

Comment:
Suggest `client_context.connected and client_context.serverless`? Or is that redundant?

Reply:
Somewhat redundant as `client_context.serverless` is set solely based on the value of `TEST_SERVERLESS` but it doesn't hurt. Updated.